### DISCUSSION TITLE
'e.split is not a function' during import

### DISCUSSION THREAD

**Discussion by PaniSPejskem** [AntlerVC/firetable#1128] [Support / Q&A] [answered]
After creating table, I tried to import data as usual. However, I am getting following error any time I select integer column to import:
![image](https://user-images.githubusercontent.com/73820773/220178440-c07f9f03-8b2a-49b3-866c-a53d033e3437.png)

I've never had any problem in my previous projects (paid or free) so it's probably something in the settings, but I cannot be sure at the moment. This is also happening on every table in that project, not only on single one.

**Comment by BeeBombshell** [comment, marked as answer]
Hey @PaniSPejskem, thanks for bringing this issue up! We have seen a few more instances of the same issue, and a fix for this will be merged into the main branch soon. You can track this PR for any updates: https://github.com/rowyio/rowy/pull/1127.
